cpp/include/raft/neighbors/detail/ivf_pq_search.cuh

```diff
@@ -712,16 +734,22 @@ inline auto get_max_batch_size(uint32_t k,
 }
 
 /** See raft::spatial::knn::ivf_pq::search docs */
+<<<<<<< HEAD
```

Comment:
I think there might be some unmerged bits here.

Reply:
Yep, I'll fix that, thanks!